### ISSUE TITLE
Remove a brush but the game object is not removed

### ISSUE THREAD

**Issue by Hisashimaru** [LogicalError/realtime-CSG-for-unity#253]
Remove a brush while selected the face, the game object is not removed. It's just removing CSGBrush component.
1. Select a face.
2. Hit a delete key.

You can get error logs in the different way. I think this is similar problem.

1. Select a face.
2. Move mouse cursor over the other face pivot point.
3. Hit a delete key.
4. Click the viewport.

![gif](https://user-images.githubusercontent.com/7605365/38479768-553b282a-3bfc-11e8-9eb6-5e386a17943c.gif)

Error logs:
```
Undo objects may not be null.
UnityEditor.Undo:RecordObjects(Object[], String)
RealtimeCSG.EditModeMeshEdit:DoMoveControlPoints(Vector3) (at Assets/Plugins/RealtimeCSG/Editor/Scripts/View/GUI/EditModeGUI/EditModes/EditMode.MeshEdit.cs:646)
RealtimeCSG.EditModeMeshEdit:HandleEvents(Rect) (at Assets/Plugins/RealtimeCSG/Editor/Scripts/View/GUI/EditModeGUI/EditModes/EditMode.MeshEdit.cs:3192)
RealtimeCSG.EditModeManager:OnSceneGUI(SceneView) (at Assets/Plugins/RealtimeCSG/Editor/Scripts/View/GUI/EditModeGUI/EditModeManager.cs:517)
RealtimeCSG.SceneViewEventHandler:OnScene(SceneView) (at Assets/Plugins/RealtimeCSG/Editor/Scripts/Control/Managers/SceneViewEventHandler.cs:54)
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr)
```

**Comment by LogicalError** [comment]
Thanks for reporting this, I'll take a look

**Comment by LogicalError** [comment]
Fix available in 1.533, I will close this ticket once this version is available from the asset store

**Comment by LogicalError** [comment]
This is available from the asset store, closing this ticket
